examples/addons/cluster-autoscaler/kubernetes-cluster-autoscaler.json

```diff
@@ -10,9 +10,16 @@
           {
             "name": "cluster-autoscaler",
             "enabled": true,
+            "pools": [
+              {
+                "name": "agentpool",
+                "config": {
+                  "min-nodes": "1",
+                  "max-nodes": "5"
+                }
+              }
+            ],
             "config": {
-              "min-nodes": "1",
-              "max-nodes": "5",
               "scan-interval": "10s"
```

Comment:
Should this get updated to match the dedault value (1m)?

Reply:
yes